apps/src/templates/sectionProgressV2/ProgressIcon.jsx

```diff
@@ -9,11 +9,11 @@ export const PROGRESS_ICON_TITLE_PREFIX = 'progressicon-';
 
 export default function ProgressIcon({itemType}) {
   const needsFeedbackTriangle = () => (
-    <div className={classNames(styles.needsFeedback, styles.cornerBox)} />
+    <div className={classNames(styles.needsFeedback, styles.cornerBox)} data-testid="needs-feedback-triangle"/>
   );
 
   const feedbackGivenTriangle = () => (
-    <div className={classNames(styles.feedbackGiven, styles.cornerBox)} />
+    <div className={classNames(styles.feedbackGiven, styles.cornerBox)} data-testid="feedback-given-triangle"/>
```

Comment:
Again, with both of these I couldn't find a single attribute that really spoke to what it was supposed to render, thus I added these in.